README.md

```diff
@@ -1 +1,68 @@
-# Space-Travelers-Hub
\ No newline at end of file
+# Space-Travelers-Hub
+- In this task, we will be working with the real live data from the SpaceX API. Our task is to build a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+# Screenshots
+
+![image](https://user-images.githubusercontent.com/87448628/146329774-2339b30f-ff2a-451f-b9a1-8c4ed53a01a5.png)
+
+![image](https://user-images.githubusercontent.com/87448628/146329869-7ff101f3-9864-4ef2-a280-248cfce284b3.png)
+
+![image](https://user-images.githubusercontent.com/87448628/146329918-5a122b3f-f6d5-4695-8791-26cf7fbc09f2.png)
+## Built With
+- React-Redux
+- CSS3
+
+### Technologies used
+- Github
+- Gitflow
+- Visual Studio Code
+## Live Demo
+
+[Link](https://cranky-cray-0feacf.netlify.app/)
+
+## Getting Started
+
+ To clone this project :
+* Run command: $git clone [repo](https://github.com/Danieldotcomcoder/space-travelers-hub)
```

Comment:
- [ ] [Optional] I would recommend you to make sure that there is command instead of link.